build.gradle

```diff
@@ -112,9 +116,11 @@ project.ext.externalDependency = [
     'kafkaAvroSerde': 'io.confluent:kafka-streams-avro-serde:5.5.1',
     'kafkaAvroSerializer': 'io.confluent:kafka-avro-serializer:5.1.4',
     'kafkaClients': 'org.apache.kafka:kafka-clients:2.3.0',
-    'logbackClassic': 'ch.qos.logback:logback-classic:1.2.9',
-    'log4jCore': 'org.apache.logging.log4j:log4j-core:2.19.0',
-    'log4jApi': 'org.apache.logging.log4j:log4j-api:2.19.0',
+    'logbackClassic': "ch.qos.logback:logback-classic:$logbackClassic",
+    'slf4jApi': "org.slf4j:slf4j-api:$slf4jVersion",
+    'log4jCore': "org.apache.logging.log4j:log4j-core:$log4jVersion",
```

Comment:
This ecosystem is still a bit convoluted to me, but why do we need all of these? Shouldn't we ideally just use logback and SLF4J? 

Also, what is the purpose of `log4j12Api`?

Reply:
Unfortunately some dependencies include the need for the older log4j api and the actual log4j impl still. It might be possible to update and/or replace those overtime however updating to latest works for now.